### PR TITLE
nerfs bdm eye crusher trophy

### PR DESCRIPTION
**Pull request by LePiromano**
## About The Pull Request
nerfs the bdm's eye to make it so instead of getting 90% resistance to damage you get 50%, doesn't change duration or stun resistance
## Why It's Good For The Game
when you're putting out 1 hit every 2 seconds and you get 1 second of what's basically immunity every hit, you end up facetanking shit that would otherwise kill you instantly.
i'm not against a "block mechanic", but it shouldn't be triggered by the main action of the weapon, perhaps whoever made it this way was trying to prevent the initial melee strike that you usually receive when getting close to the megafauna, but there's already a trophy that prevents megafauna from striking with a melee too fast (the lobster claw, when it decides to work).
what this trophy ends up doing ingame, for most miners, is accidentally saving them from things that they should have not survived, teaching them nothing about any error they made, and instead just rewarding them with their powergame loot just by mindlessly hitting the big demon with an axe.

## Changelog
:cl: John Mining
balance: Nerfed the Blood-Drunk Miner's crusher trophy to only apply a 50% damage resistance for 1 second.
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
In an ideal world, you are making an attack every 0.8 seconds. You also have to make a crusher shot between these, which also puts you on attack cooldown for the same duration. If you have the skull trophy, that's 1.1 seconds, and 1.4 seconds otherwise. for the internal cooldown of the crusher (which, interestingly, doesn't seem to meaningfully do much for actual fighting as a result).

Since the crusher itself cannot consistently maintain the buff without creating at least some window, there is a chance you may be hit during openings in the ideal situation where you're simply swinging and hitting marks at every opportunity you get. The actual click cooldowns themselves could be manipulated in some way to speed them up, but the crusher itself cannot exceed that 1.1 second timing. There will always be a window of vulnerability.

This is also an ideal scenario. In reality, we have plenty of boss monsters that do a bunch of rapid movements or fling you about. So the timings aren't quite as good as this makes them out to be. Mobs won't necessarily let you do all this facetanking unless you're carrying around, say, a dead mob to proc marks with. Which, in that case, would be maybe worth fixing in its own right.

All this being said, I don't think people particularly go chasing for this trophy as it is because of the inconsistency of the buff. It is nice when it works but you still need to put in some legwork usually to make it strong. I don't believe it is as braindead as you make it out to be.

**TL;DR** I think you may be exaggerating the benefits of the defensive buff.

**Comment by DrAmazing343** [comment]
The idea of 1 hit per 2 seconds for 1 second immunity is a little flawed for the average miner; 50% DR is still plenty, but honestly, the uptime on this is far too short to justify nerfing it. There's still lots of attacks that you 100% want to try and avoid rather than take even 10% of the damage, and plenty of ranged attacks/attacks so fast that this can't even proc, such as Ash Drake's fire blast, and Bubblegum's rushdown attack(s!!).

I don't see this as a necessary or worthwhile change at all, especially since it's purely a PvE trophy, and one that the experts don't need or rely on anyway. This is a trophy off of one of the two beginner boss fights that especially helps beginners, even if it teaches them SOME bad habits— I'd not like to see that be diminished, especially since there's no real way to powergame through the trophy itself.

**Comment by necromanceranne** [comment]
Ha, I got my giant wall of text in first, that makes me superior.

**Comment by LePiromano** [comment]
if the trophy was coming from a normal megafauna with 2500 health and insane damage i wouldn't have bothered in the first place, this shit comes from a megafauna which you can face tank with 3 goliath plates on your armor and a survival pen.
so responding to your concerns more directly:
"exaggerating the defensive buff"
you can eat 10 death bolts point blank from a colossus to just take 25 damage
you can eat a quadruple charge from bubblegum head on and take around 30 damage
sometimes this shit actually happens and you don't even realize it because it's not an active thing, i've had times where i finished killing a colossus and when i look in the chat i realized at 3 points in the fight i just ate like a retribution point blank and nothing happened to me, i wouldn't blame you if you didn't realize just how strong this trophy is because there's no feedback for when you just tank 90% of an attack that would be otherwise death

"the uptime is too short"
true but it's also pretty easy to just sync your hits with the attacks of a megafauna, it's almost the same timing, try it, sometimes you even do it on accident

"it is a pve trophy"
yeah i really just want more miners to die

i'm not talking out of my ass i have 4844 boss kills (which in reality makes my opinion completely invalid), but i'll leave it up to mantainers to determine what they want to do with this, also tell me if i fucked something up code wise this is my first pr

**Comment by Namwty** [comment]
I don't see how "I want miners to die more often" is good for the game, but regardless.

If you have 4844 kills or whatever, then you should already know, that people who can kill colossus hundred times, will be able to do colossus with or without this trophy. Arguably this will only hurt people who are new, that's about it, which isn't exactly good for the game either.

If you tell me that **you** suddenly will turn into a noobie and won't be able to kill anything and "die more often" if they nerf it, then I don't even know lol.


And if someone wants more challenging experience, they are free to play with blind quirk and whatever else.

**Comment by LePiromano** [comment]
if it wasn't clear already the "i want miners to die" part is a joke, my main problem with the bdm's trophy is that it saves you from things that would otherwise kill you instantly, without you even noticing, keep in mind 50% is still a ton, but not enough to make it so if you eat 6 death bolts to the face you wont even feel it.
a new miner shouldn't just randomly beat the colossus on their first try when in reality they would've died 7 times through the fight, that's not teaching them anything (altough a new miner shouldn't be fighting a colossus in the first place)

**Comment by the-orange-cow** [comment]
The trophy is designed to give you a shot at surviving megafauna when going into melee range, which is is kind of a requirement for using a melee weapon.

Also your wigftg isn't very good.

**Comment by LePiromano** [comment]
the whole point of the crusher is to be a riskier version of the pka, that said risk is that you have to get close to the megafauna, removing that risk for an entire second is pretty crazy, specially when you do it every single attack

seeing as a lot of people aren't content with my solution:
would it be better if the trophy rewarded you with a smaller, but constant stackable damage reduction? let's say 30%, all the time, as long as you can keep landing consistent hits within 3 seconds of the last one :D

**Comment by DrAmazing343** [comment]
> the whole point of the crusher is to be a riskier version of the pka, that said risk is that you have to get close to the megafauna, removing that risk for an entire second is pretty crazy, specially when you do it every single attack
> 
> seeing as a lot of people aren't content with my solution: would it be better if the trophy rewarded you with a smaller, but constant stackable damage reduction? let's say 30%, all the time, as long as you can keep landing consistent hits within 3 seconds of the last one :D

I think rather than try to figure out an alternative "solution," we should explore the "problem" this PR is trying to solve. The Crusher is riskier by nature, and this Trophy appears to be very specifically to assist newer Miners who are still bothering with BDM to have a chance at learning more Megafauna v. Crusher fights. I don't see a problem with any Miners "gaming" this at all, and I don't think we should nerf this at all.

I believe your experience at mining has made this out to be an issue, as you see it as just a crutch, or something, but it is a genuinely useful thing for newer Miners to have to cover their asses as they learn difficult fights. We don't need to worsen the new Miner cliff that starts at the Megafauna.

**Comment by LePiromano** [comment]
like i said, i don't think a new miner would learn anything from surviving a fight where they would've normally died 5 times without the trophy, at that point you're just making it so the players think the trophy is necessary to fight the megafauna at all.

i'm not actively TRYING to get them to die, i would've said decrease it to like 20% resistance if i was, 50% resistance is still enough to make it so you can survive most of the megafauna instakill attacks, but without walking out of it without a scratch.

this way little jimmy miner will understand that he just ate shit big time, but still wont be punished with a shift removal by it, unless he was fighting the megafauna at already low health, which is something little jimmy must learn not to do

**Comment by DrAmazing343** [comment]
> like i said, i don't think a new miner would learn anything from surviving a fight where they would've normally died 5 times without the trophy, at that point you're just making it so the players think the trophy is necessary to fight the megafauna at all.
> 
> i'm not actively TRYING to get them to die, i would've said decrease it to like 20% resistance if i was, 50% resistance is still enough to make it so you can survive most of the megafauna instakill attacks, but without walking out of it without a scratch.
> 
> this way little jimmy miner will understand that he just ate shit big time, but still wont be punished with a shift removal by it, unless he was fighting the megafauna at already low health, which is something little jimmy must learn not to do

Little Jimmy learns by fighting the Megafauna. The more time he's in the fight, the more he'll learn to avoid the attacks. If he wins three times WITH the Trophy, I'd argue he's learned more than five times DYING without it. Learning is not negated by an advantage.

**Comment by projectkepler-ru** [comment]
How in the fuck is "I want miner to die" a valid reason to have this
Do you realise that this doesn't work against carbon-based player???

**Comment by improvedname** [comment]
you cant make megafauna more intense unless it has bullshit RNG, lets take the concept of megaman

Robot masters might be challenging at first but once you learn their pattern they kinda become a joke and you can easily clear it not taking damage.

For example crashman if you jump he jumps high and shoots a bomb at you once you learn that you just kinda perfect the fight, trophies just exist as a safety net

**Comment by CCC23** [comment]
as a person who has played alot of miner im going to be real with you this effect could have been completly bugged and nonfunctional and I think it would take most miners atleast 2 weeks to actually realize something has changed at all most of the time a miner isnt thinking oh I should try to get this hit in so I can tank this hit. The more realistic approach is more along the lines of the wolf ear trophey of I GOTTA RUN so you try to get out inbetween the attack cooldown of the crusher. Anyone and I mean ANYONE who even trys to face tank a boss will ultimately learn that attempting to face tank any boss (beside blood drunk miner) will eventually just end with you getting hit with either a piercing attack a instakill(bubblegums rushdown) or a shotgun blast to the face(collosus that lagged while it was shooting so it shot 40 shotgun blast deathbolts directly into your skull)

**Comment by LePiromano** [comment]
well seeing as my approach isn't the most agreed upon i'll close this, i'll see if i can figure out some way of reworking the trophy to make the "block action" more noticeable and untie it from the main action of the crusher, because it is a neat idea to have some kind of point blank block mechanic, just not when it's triggered by using the weapon normally, it's like having your main attack also be a dodge roll in a darksouls game
but hey at least i learnt how to open a pr :DD
